src/OrbitQt/CaptureOptionsDialog.h

```diff
@@ -48,7 +48,7 @@ class CaptureOptionsDialog : public QDialog {
 
   void SetCollectMemoryInfo(bool collect_memory_info);
   [[nodiscard]] bool GetCollectMemoryInfo() const;
-  void SetMemorySamplingPeriodNs(uint64_t memory_sampling_period_ns);
+  void SetMemorySamplingPeriodMs(uint64_t memory_sampling_period_ns);
```

Comment:
This does not make much sense. The method has the suffix `Ms` and the parameter is still `ns`. The implementation also still wants a nanosecond value.

Reply:
Thanks for pointing out this issue! 

We are using nanoseconds in the proto but we would like to only allow the user to set the sampling period in milliseconds in the ui. So I change to rename some components and methods by replacing "Ns" with "Ms".

I change the component in the ui to be `memorySamplingPeriodMsLineEdit`. And here when setting the sampling period programmatically, we pass the value in nanoseconds and this `SetMemorySamplingPeriodMs` method will convert it to milliseconds and then set up the value in the ui.

When getting the sampling period value in the ui with `GetMemorySamplingPeriodNs`, we get the raw value in milliseconds and then convert it to nanoseconds.

Reply:
Sorry, I dont think I was clear enough with what I meant. From an api design standpoint you want to design and name your methods in a way that it is clear how to use them. 

If I as a user read the name `SetMemorySamplingPeriodMs`, I assume I can give this a value in milliseconds and it sets the memory sampling period. But this is not the case. I actually have to give a value in nanoseconds. This is confusing for someone who wants to use that method.

Reply:
Thanks for the explanation! I changed to use only milliseconds in the CaptureOptionsDialog .h/.cpp/.ui files. 

The conversions to nanoseconds are moved to the OrbitMainWindow.

Reply:
Thanks!!